docs(Guides): Added guidance for hosting internally (#947)
carbon-design-system/gatsby-theme-carbon: packages/example/src/pages/guides/hosting.mdx

---
title: Hosting
description:

>   Overview of different options IBMers have to host sites built using the Carbon

Reviewer: ```suggestion
  Overview of different options for hosting sites built using the Carbon
```

  Gatsby theme
---

<PageDescription>

> For IBMers building sites using the Carbon Gatsby Theme, there are number of
> different ways you can host your site internally. While the Carbon team does not
> support hosting in and of itself. We are are happy to point you in the right
> direction for your hosting needs.

Reviewer: ```suggestion
While the Carbon team does not have an opinionated stance on how you host your projects, these are some resources others have found helpful for hosting.
```

</PageDescription>

## GH Pages

The CIO provides
[Github Enterprise](https://w3.ibm.com/help/#/article/github_ent_ibm/github_overview?requestedTopicId=github_overview)
to IBMers for source code control while working with IBM intellectual property.
This tool is accessible by any IBMer. Check out the guide below from Gatsby to

> allow your site to work with GH Pages.

Reviewer: Can update this to not be IBM specific maybe

---
title: Hosting
description:
  Overview of different hosting options for sites built using the Carbon Gatsby
  theme
---

<PageDescription>

When building sites using the Carbon Gatsby Theme, there are a number of
different ways you can host your site. While the Carbon team does not have an
opinionated stance on how you host your projects, these are some resources
others have found helpful for hosting.

</PageDescription>

## GH Pages

> For external sites, you can use [GH Pages](https://pages.github.com/) on public
> Github to build your site. For internal sites, the CIO provides
> [Github Enterprise](https://w3.ibm.com/help/#/article/github_ent_ibm/github_overview?requestedTopicId=github_overview)
> to IBMers while working with IBM intellectual property. This tool is accessible
> by any IBMer. Regardless of whether your projects is external or internal, check
> out the guide below from Gatsby to allow your site to work with GH Pages.

Reviewer: ```suggestion
[GitHub Pages](https://pages.github.com/) pages is a simple, low maintenance approach to get your site hosted quickly. For internal sites, or sites with proprietary information, IBMers can leverage [Github Enterprise](https://w3.ibm.com/help/#/article/github_ent_ibm/github_overview?requestedTopicId=github_overview). Regardless of whether your projects is external or internal, there are a few steps (see below) to allow your site to work with GitHub Pages.
```
